astropy/io/fits/hdu/base.py

```diff
@@ -275,24 +275,21 @@ def fromstring(cls, data, checksum=False, ignore_missing_end=False,
         Parameters
         ----------
         data : str, bytearray, memoryview, ndarray
-           A byte string containing the HDU's header and data.
-
```

Comment:
Why remove blank line?

Reply:
I think this snuck in because there was also and indentation level fix - I'll revert the whitespace changes here.